src/objects/aff/zcl_abapgit_json_handler.clas.abap

```diff
@@ -237,40 +235,28 @@ CLASS zcl_abapgit_json_handler IMPLEMENTATION.
     DATA: lt_st_source      TYPE abap_trans_srcbind_tab,
           lo_mapping        TYPE REF TO zif_abapgit_ajson_mapping,
           lv_json           TYPE string,
-          lo_ajson          TYPE REF TO zcl_abapgit_ajson,
-          lo_ajson_filtered TYPE REF TO zif_abapgit_ajson,
+          lo_ajson          TYPE REF TO zif_abapgit_ajson,
           lo_filter         TYPE REF TO lcl_aff_filter.
 
     FIELD-SYMBOLS: <lg_source> LIKE LINE OF lt_st_source.
 
     APPEND INITIAL LINE TO lt_st_source ASSIGNING <lg_source>.
     GET REFERENCE OF iv_data INTO <lg_source>-value.
 
-    lo_mapping = zcl_abapgit_ajson_mapping=>create_camel_case( iv_first_json_upper = abap_false ).
-
-    lo_ajson = zcl_abapgit_ajson=>create_empty( ii_custom_mapping = lo_mapping ).
-
-    lo_ajson->keep_item_order( ).
-    lo_ajson->set(
-      iv_path = '/'
-      iv_val  = iv_data ).
+    lo_ajson = zcl_abapgit_ajson=>new( iv_keep_item_order = abap_true
+      )->set( iv_path = '/'
+              iv_val  = iv_data
+      )->map( zcl_abapgit_ajson_mapping=>create_to_camel_case( ) ).
 
     map2json_original_language( CHANGING co_ajson = lo_ajson ).
     map2json_abap_language_version( CHANGING co_ajson = lo_ajson ).
     map2json_custom_enum( EXPORTING it_enum_mappings = iv_enum_mappings
                           CHANGING co_ajson          = lo_ajson ).
 
     CREATE OBJECT lo_filter EXPORTING iv_skip_paths = iv_skip_paths.
-    lo_ajson_filtered = zcl_abapgit_ajson=>create_from(
-                          ii_source_json = lo_ajson
-                          ii_filter      = lo_filter ).
-
-    lo_ajson_filtered->keep_item_order( ).
```

Comment:
Is `keep_item_order` the default in the new API? Or can we skip this?

Reply:
actually, in the old version this was a 'bug' ( `create_from` did not preserved the keep_item_order setting ). So, this is an intentional change.